example/HtmlRunnerExample.elm

```diff
@@ -11,11 +11,12 @@ import String
 import Expect
 import Test exposing (..)
 import Test.Runner.Html
+import Test.Runner.Html.App
 import Fuzz exposing (..)
 import Char
 
 
-main : Program Never
+main : Program Never (Test.Runner.Html.App.Model Test.Runner.Html.Msg Test.Runner.Html.Model) (Test.Runner.Html.App.Msg Test.Runner.Html.Msg)
```

Comment:
See `TestProgram` in node-test-runner (i.e. https://github.com/rtfeldman/node-test-runner/commit/709dbed8f85f2b43c82e383526dfdfcfcec05a0c)